numba/cuda/testing.py

```diff
@@ -31,8 +39,11 @@ class ContextResettingTestCase(CUDATestCase):
     to be internal implementation details (such as the state of allocations and
     deallocations, etc.).
     """
+    def setUp(self):
+        super().setUp()
```

Comment:
I don't think this is necessary, because `ContextResettingTestCase.setUp()` isn't doing anything.